vignettes/volcalc.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Introduction to volcalc"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Introduction to volcalc}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(volcalc)
+```
+
+The primary function in `volcalc` is `calc_vol()`.  It accepts either a path to a .mol file or a SMILES string.  There are a few example .mol files included in the `volcalc` installation.  The paths to these can be found with `mol_example()`
+
+### Basic usage with .mol files
+
+```{r}
+#list possible mol files
+mol_example()
+
+#get path to specific file
+mol_path <- mol_example("C16181.mol")
+mol_path
+```
+
+The default output of `calc_vol()` includes a relative volatility index, `rvi` which is equivalent to $\textrm{log}_{10}C^\ast$.  It also includes a RVI category for clean air.
+
+```{r}
+calc_vol(mol_path)
+```
+
```

Comment:
If the categories are not defined anywhere else, like the README, I think the default ones should be defined here?

Reply:
The categories are defined in the help file for `calc_vol()` under the `environment` parameter.  But we need to better document the columns output by `calc_vol()` and I'll do that in a different PR